### PR TITLE
removed twitter method #603

### PR DESCRIPTION
**Pull request by sarthakw7**
## Pull request checklist

- [ ] My code contains tests covering the code I modified
- [ ] I linted and tested the project with `bin/check`
- [x] I added significant changes and product updates to the [changelog](CHANGELOG.md)

### CODE REVIEW

File: app/components/developers/open_graph_tags_component.rb [hunk 1]
```diff
@@ -25,8 +25,5 @@ def twitter_card
       "summary"
     end
```
Comment (joemasilotti) [inline app/components/developers/open_graph_tags_component.rb:27]: Please remove this blank line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ## August
 
+* August 19 - Removed Twitter method from developer Open Graph tags #603
```
Comment (joemasilotti) [inline CHANGELOG.md:7]:
```suggestion
* August 19 - Removed developer Twitter handle from Open Graph tags #607
```

**Review by joemasilotti — CHANGES_REQUESTED** [review]
Looks great! I made two small requests then we are good to merge.

**Comment by sarthakw7** [comment]
Sure .

**Comment by joemasilotti** [comment]
Thanks! It looks like there are some test failures though.

I think you can remove line 23 in `test/components/developers/open_graph_meta_tags_component_test.rb` and the entire test on line 35.

**Review by joemasilotti — APPROVED** [review]
Looks great – thanks for the contribution!

**Comment by sarthakw7** [comment]
Glad to help!

**Pull request merged**